<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtcSetDialog</class>
 <widget class="QDialog" name="RtcSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTC SET PROPERTY</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>840</width>
     <height>604</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelRtcTitle">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>790</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>RTC Battery Error</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetRestart">
    <property name="geometry">
     <rect>
      <x>417</x>
      <y>490</y>
      <width>400</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string> Set &amp;&amp; Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>305</y>
      <width>790</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeRtc">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>330</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRtcStatus">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>145</y>
      <width>790</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Please replace the battery and set time.</string>
    </property>
   </widget>
   <zorder>labelRtcTitle</zorder>
   <zorder>buttonSetRestart</zorder>
   <zorder>labelRtcStatus</zorder>
   <zorder>label</zorder>
   <zorder>dateTimeRtc</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
